specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Trigger.json

```diff
@@ -365,6 +372,24 @@
           "maximum": 86400
         }
       }
+    },
+    "TumblingWindowDependency": {
+      "description": "Tumbling Window dependency information.",
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "Reference type \"TriggerReference\" for Tumbling Window.",
+          "type": "string"

```

Comment:
I think ARM folks will comment on this too, you should add an enum here:

"enum": [
    "TumblingWindowTriggerReference"
]

Reply:
Resolved